### PR TITLE
change REACT_APP prefixes to IMJS

### PR DESCRIPTION
**Pull request by kckst8**

### CODE REVIEW

File: packages/modules/cra-template-itwin-viewer/template/.env [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # ---- Authorization Client Settings ----
-REACT_APP_AUTH_CLIENT_CLIENT_ID=imodeljs-spa-samples-2686
-REACT_APP_AUTH_CLIENT_REDIRECT_URI=http://localhost:3000/signin-callback
-REACT_APP_AUTH_CLIENT_LOGOUT_URI=http://localhost:3000/logout
-REACT_APP_AUTH_CLIENT_SCOPES ="openid email profile organization imodelhub context-registry-service:read-only product-settings-service general-purpose-imodeljs-backend imodeljs-router"
+IMJS_AUTH_CLIENT_CLIENT_ID=imodeljs-spa-samples-2686
+IMJS_AUTH_CLIENT_REDIRECT_URI=http://localhost:3000/signin-callback
+IMJS_AUTH_CLIENT_LOGOUT_URI=http://localhost:3000/logout
+IMJS_AUTH_CLIENT_SCOPES ="openid email profile organization imodelhub context-registry-service:read-only product-settings-service general-purpose-imodeljs-backend imodeljs-router"
```
Comment (aruniverse) [inline packages/modules/cra-template-itwin-viewer/template/.env:5]: we should probably add the scope `urlps-third-party`

**Review by aruniverse — APPROVED** [review]

**Comment by aruniverse** [comment]
Actually one minor comment, did you test this out? I'm not sure if the env var is case sensitive or not. It looks like we support `imjs_`  not `IMJS_`, but could be wrong about that

File: packages/modules/cra-template-itwin-viewer/template/.env (hunk 1 above)
Comment (kckst8) [reply packages/modules/cra-template-itwin-viewer/template/.env:5]: what is that scope for?

**Comment by kckst8** [comment]
> Actually one minor comment, did you test this out? I'm not sure if the env var is case sensitive or not. It looks like we support `imjs_` not `IMJS_`, but could be wrong about that

I did test and it worked fine. I think we should fall in line with CRA (and node env vars in general) and keep them UCase

File: packages/modules/cra-template-itwin-viewer/template/.env (hunk 1 above)
Comment (kckst8) [reply packages/modules/cra-template-itwin-viewer/template/.env]: added

**Pull request merged**